src/main/java/seedu/address/logic/commands/DeleteMultipleCommand.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Deletes a multiple persons identified using the displayed index range from the address book.
+ */
+public class DeleteMultipleCommand extends Command {
+    public static final String COMMAND_WORD = "deletem";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes a range of persons identified by the index number used in the displayed person list.\n"
+            + "Parameters: START_INDEX - END_INDEX (must be positive integers)\n"
```

Comment:
not fully sure how to phrase it, but it might be useful to specify that the indices are inclusive?